pkg/discovery/discovery_manager.go

```diff
@@ -1,4 +1,6 @@
 // Copyright 2022 The Parca Authors
+// TODO: This license is not consistent with license used in the project.
+//       Delete the inconsistent license and above line and rerun pre-commit to insert a good license.
```

Comment:
Never fight a bot, it does not get tired :laughing:

The TODOs from 2022 → 2023 → 2022 pass would need to be cleaned up, so the check can succeed

The autofix was added recently to assist contributors with style nits, if the majority thinks it does not help, I have no strong opinion for keeping it, although it only seems to require a bit of tuning/fixing here

Reply:
I wrote those when I was frustrated with the bot. We can keep it if we tune it. 
Let's give it more chance ☺️ 

I don't want them to fight against me when Skynet has risen.